website/docs/docs/dbt-versions/2024-release-notes.md

```diff
@@ -20,6 +24,8 @@ dbt Cloud release notes for 2024.
   <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-create-merge-job.png" width="90%" title="Example of creating a merge job"/>
 
   </expandable>
+  
+- **Behavior change:** Introduced the [`require_explicit_package_overrides_for_builtin_materializations`](/docs/reference/global-configs/legacy-behaviors#require_explicit_package_overrides_for_builtin_materializations) flag, opt-in and disabled by default. If set to `True`, dbt will only use built-in materializations defined in the root project or within dbt, rather than implementations in packages. This will become the default in May 2024 (dbt Core v1.8 and dbt Cloud "Keep on latest version").
```

Comment:
```suggestion
- **Behavior change:** Introduced the [`require_explicit_package_overrides_for_builtin_materializations`](/reference/global-configs/legacy-behaviors#require_explicit_package_overrides_for_builtin_materializations) flag, opt-in and disabled by default. If set to `True`, dbt will only use built-in materializations defined in the root project or within dbt, rather than implementations in packages. This will become the default in May 2024 (dbt Core v1.8 and dbt Cloud "Keep on latest version").
```